Apps/FlightSoftware/Watchdog/src/main.c

```diff
@@ -20,11 +20,14 @@ extern uint8_t heating;
 uint8_t lastHeater = 0;
 uint8_t heatingControlEnabled = 1;
 
+I2C_Sensors__Readings i2cReadings;

```

Comment:
If we're going to have the i2cReadings in a global anyways, why do we pass it on to `send_earth_heartbeat`? I'm just not a big fan of passing this large struct on the stack.

Reply:
Ah so I started with it global and then switched to passing it as an arg. I am fine with whichever, just let me know which is better and I'll make it consistent (sounds like you prefer global)

Reply:
I generally lean against globals whenever possible, but if you keep this as a global it should declared `static`

Reply:
Removed global, now just passing pointer to i2cReadings as arg in functions where it is needed